opentreemap/importer/views.py

```diff
@@ -110,6 +113,18 @@ def get_import_table(request, instance, table_name):
 _EVENT_TABLE_PAGE_SIZE = 5
 
 
+def _cleanup_tables(instance):
+    ievents = (list(TreeImportEvent.objects.filter(instance=instance)) +
+               list(SpeciesImportEvent.objects.filter(instance=instance)))
+    for ie in ievents:
+        if ie.task_id != '':
```

Comment:
Maybe not necessary because they would not produce a truthy value for `AsyncResult(ie.task_id).failed()`, but can't hurt, so I'll add a check to only look at import events that are not in finished states.